sources/packages/backend/apps/queue-consumers/src/services/student-file/student-file.service.ts

```diff
@@ -49,7 +55,14 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
         UNABLE_TO_SCAN_FILE,
       );
     }
-    await this.updateFileScanStatus(studentFile.uniqueFileName, isInfected);
+    if (isInfected) {
+      processSummary.warn("Virus found.");
+      processSummary.info("Updating infected file name and status.");
+      await this.updateInfectedFile(studentFile);
+    } else {
+      processSummary.info("No virus found.");
+      await this.updateFileScanStatus(studentFile.uniqueFileName, isInfected);
```

Comment:
Probably from a previous PR but it does not seem accurate to the method `updateFileScanStatus` to receive the `isInfected` as a `boolean` and the switch check for `null` or `undefined`.